notebook/notebook/notebook.py

```diff
@@ -157,6 +157,9 @@ def wrap(fun):
             limits={'cpu': cpu, 'memory': memory})
 
     pod_spec = kube.client.V1PodSpec(
+        node_selector={
+            'preemptible': 'false'
```

Comment:
here we're using a node selector instead of a toleration?

Reply:
Mistake.  Good catch!  Fixed.